quill-core/src/main/scala/io/getquill/quotation/ReifyLiftings.scala

```diff
@@ -41,8 +41,8 @@ trait ReifyLiftings {
 
         case Property(CaseClassValueLift(name, v: Tree), prop) =>
           val term = TermName(prop)
-          val tpe = v.tpe.member(term).typeSignatureIn(v.tpe)
-          val merge = c.typecheck(q"$v.$term")
+          val merge = q"$v.$term"
+          val tpe = c.typecheck(q"import language.reflectiveCalls; $merge").tpe
```

Comment:
NullPointerException is thrown because `v` is not typed tree and `v.tpe` is `null`. `c.typecheck(q"$v.$term")` is also fails because `liftings` is [defined as a structural type](https://github.com/mxl/quill/blob/314e8b4cbef40689f7ece32799630a189e22481f/quill-core/src/main/scala/io/getquill/quotation/ReifyLiftings.scala#L93) and `import language.reflectiveCalls` is needed to access its members.